torchaudio/datasets/speechcommands.py

```diff
@@ -22,6 +22,15 @@
 }
 
 
+def _load_list(root, *filenames):
+    output = []
+    for filename in filenames:
+        filepath = os.path.join(root, filename)
+        with open(filepath) as fileobj:
+            output += [os.path.normpath(os.path.join(root, line.strip())) for line in fileobj]
```

Comment:
Can you add a comment on why `normapth` is necessary here?
Even though [`normpath`'s official document](https://docs.python.org/3/library/os.path.html#os.path.normpath) says `On Windows, it converts forward slashes to backward slashes. To normalize case, use normcase().`. For many of us, who are linux users, It is not well known. (I learned the only today.)